src/components/button/styles.css

```diff
@@ -0,0 +1,72 @@
+.button {
+  height: auto;
+  width: auto;
+  padding: 10px 25px;
+  border-radius: 16px;
+  border: none;
+  background-color: white;
+  text-align: center;
+  font-family: Montserrat, sans-serif;
+  font-size: 2.6rem;
+  font-style: normal;
+  font-weight: 280;
+  box-shadow: 0px 5px 5px rgba(0, 0, 0, 0.35);
+  transition-property: background-color;
+  transition-duration: 700ms;
+  min-width: max-content;
+  max-width: 100%;
+}
+
+.default {
+  grid-area: content;
+  position: relative;
+}
+
+.hof {
+  grid-area: mode-description;
+  display: inline-flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  align-items: center;
+}
+
+.hof img {
+  width: 32px;
+  height: 32px;
+  margin: auto 10px auto 0;
+}
+
+.ptg {
+  background-color: red;
+  color: white;
+  grid-area: mode-description;
+}
+
+.button:hover {
+  box-shadow: 0 0 20px rgba(255, 0, 0, 0.9);
+}
+
+.ptg:hover {
+  background-color: rgb(201, 3, 3);
+}
+
+@media only screen and (max-width: 660px) {
+  button.button {
```

Comment:
.button is enough I think. You dont' need to have htmltag and class.

below the same

Reply:
I did it the way you say, but it didn't do anything because the selector's specifity was too general.